doc/fileformat/serdes.rst

```diff
@@ -125,20 +125,20 @@ the relevant items. The ``begin_write_array`` and ``end_write_array`` methods
 bracket a write of an array value and announce that the following keys are to
 be interpreted as integer indices. Analogous for the ``map`` counterparts and
 the associated ``begin_read`` and ``end_read`` methods. Finally, ``next_key`` is
-used during reading of containers to retrieve an optional next key and advanced
+used during the reading of containers to retrieve an optional next key and advances
 the internal iterator. If empty, the container is exhausted, else the contained
-key can be used to retrieve the associated value. See examples below and the JSON
+key can be used to retrieve the associated value. See the examples below and the JSON
 interface in ``arborio``.
 
 
 Adding Snapshotting to new Objects (C++ only)
 ---------------------------------------------
 
-This is not available at the Python interface, due to a mismatch in features at
-the level of languages and binings generation.
+This is not available at the Python interface due to a mismatch in features at
+the level of languages and bindings [is this correct?] generation.
```

Comment:
Yes, we cannot add an object to the serializer without having access to its C++ definition.

Reply:
fixed it